<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sync</class>
 <widget class="QMainWindow" name="sync">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bestsync</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../logo (2).png</normaloff>../logo (2).png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="11">
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Synchronize</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../sync.png</normaloff>../sync.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="11">
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="9">
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sync after an hour</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string extracomment="File time &amp; size">Compare</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../file.png</normaloff>../file.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="toolButton_5">
      <property name="toolTip">
       <string>left to right sync</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../right.jpg</normaloff>../right.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QToolButton" name="toolButton_7">
        <property name="toolTip">
         <string>Comparison setting</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../settings (1).png</normaloff>../settings (1).png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_8">
        <property name="toolTip">
         <string>Filter</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../funnel.png</normaloff>../funnel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_6">
        <property name="toolTip">
         <string>Synchronization setting</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../settings.png</normaloff>../settings.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="9">
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButton_4">
      <property name="toolTip">
       <string>right to left sync</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../left.jpg</normaloff>../left.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="9" rowspan="2" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="treeView"/>
      </item>
      <item>
       <widget class="QTreeView" name="treeView_2"/>
      </item>
     </layout>
    </item>
    <item row="1" column="10">
     <widget class="QToolButton" name="toolButton_10">
      <property name="toolTip">
       <string>Swap</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../swap.png</normaloff>../swap.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_as_batch_job"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionStart_comparison_2"/>
    <addaction name="separator"/>
    <addaction name="actionComparison_settings_2"/>
    <addaction name="actionFilter_settings"/>
    <addaction name="actionSynchronization_settings"/>
    <addaction name="separator"/>
    <addaction name="actionStart_synchronization"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="action"/>
    </widget>
    <addaction name="actionFind"/>
    <addaction name="actionExport_file_list"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionView_help"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuAction"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../new.png</normaloff>../new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../open-file-button.png</normaloff>../open-file-button.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../save.png</normaloff>../save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart_comparison_2">
   <property name="icon">
    <iconset>
     <normaloff>../file.png</normaloff>../file.png</iconset>
   </property>
   <property name="text">
    <string>Start comparison</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionComparison_settings_2">
   <property name="icon">
    <iconset>
     <normaloff>../settings (1).png</normaloff>../settings (1).png</iconset>
   </property>
   <property name="text">
    <string>Comparison settings</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionFilter_settings">
   <property name="icon">
    <iconset>
     <normaloff>../funnel.png</normaloff>../funnel.png</iconset>
   </property>
   <property name="text">
    <string>Filter settings</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionSynchronization_settings">
   <property name="icon">
    <iconset>
     <normaloff>../settings.png</normaloff>../settings.png</iconset>
   </property>
   <property name="text">
    <string>Synchronization settings</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionStart_synchronization">
   <property name="icon">
    <iconset>
     <normaloff>../sync.png</normaloff>../sync.png</iconset>
   </property>
   <property name="text">
    <string>Start synchronization</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionLanquage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset>
     <normaloff>../binoculars.png</normaloff>../binoculars.png</iconset>
   </property>
   <property name="text">
    <string>Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionExport_file_list">
   <property name="text">
    <string>Export file list</string>
   </property>
  </action>
  <action name="actionReset_layout">
   <property name="text">
    <string>Reset layout</string>
   </property>
  </action>
  <action name="actionView_help">
   <property name="icon">
    <iconset>
     <normaloff>../faq.png</normaloff>../faq.png</iconset>
   </property>
   <property name="text">
    <string>View help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../customer-service.png</normaloff>../customer-service.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_as_batch_job">
   <property name="icon">
    <iconset>
     <normaloff>../terminal.png</normaloff>../terminal.png</iconset>
   </property>
   <property name="text">
    <string>Save as batch job</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset>
     <normaloff>../united-kingdom.png</normaloff>../united-kingdom.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>../iran.png</normaloff>../iran.png</iconset>
   </property>
   <property name="text">
    <string>فارسی</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
